actions/createChainsAndERC20List.js

```diff
@@ -3,17 +3,45 @@ import path from 'path';
 import { ROOT_PATH, ERC20_TOKEN_LIST_FILE } from './constants.mjs';
 import { getTokens } from './getTokens.mjs';
 import { createChain } from './createChain.mjs';
+import { getAverageColor } from 'fast-average-color-node';
+import async from 'async';
 
-const chains = fs.readdirSync(ROOT_PATH).reduce((acc, chainId) => {
-  // this is all chain paths. ie.../subnet-assets/chains/11111
-  const chainTokenIds = path.resolve(ROOT_PATH, chainId);
+fs.readdir(ROOT_PATH, async (err, files) => {
+  if(!err && files) {
+    let chains = {};
 
-  const chain = {
-    ...createChain(chainId, chainTokenIds),
-    tokens: getTokens(chainId, chainTokenIds).filter((token) => token.contractType === 'ERC-20'),
-  };
+    await async.forEachOf(files, async (chainId) => {
+      // this is all chain paths. ie.../subnet-assets/chains/11111
+      const chainTokenIds = path.resolve(ROOT_PATH, chainId);
+      const tokens = getTokens(chainId, chainTokenIds);
+      const tokensWithAverageColor = await async.map(tokens, async (token) => {
+        if (token.logoUri) {
+          try {
+            const color = await getAverageColor(token.logoUri);
+            
+            return {
+              ...token,
+              tokenColor: color.hex,
+            };
+          } catch (err) {
+            console.log(err);
+          }
+        }
+        return token;
+      });
```

Comment:
Nice!

Can we abstract this into a method in a separate `createToken` file (similar to how we have the `createChain` file) for the sake of consistency?